### ISSUE TITLE
Accessibility missing from filter radio button group on devtools-bug-finder

### ISSUE THREAD

**Issue by medeaoblongata** [captainbrosset/devtools-bug-finder#40]
Now that a beta version of the Accessibility tool is going out with the formal FF releases, it would be convenient and make sense to have a filter on devtools-bug-finder for this tool.
